test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatConfigTests.cs

```diff
@@ -128,12 +128,12 @@ public void ConfigGetCommand_ConfigKeyArgNoDirectoryArg_Success()
         public void ConfigGetCommand_AllArg_Success()
         {
             // Arrange & Act
-            using var testInfo = new TestInfo("NuGet.Config", "/subfolder");
+            using var testInfo = new TestInfo("NuGet.Config", "\\subfolder");
             {
                 var result = CommandRunner.Run(
                     DotnetCli,
                     Directory.GetCurrentDirectory(),
-                    $"{XplatDll} config get all {testInfo.WorkingPath + "/subfolder"}",
+                    $"{XplatDll} config get all {testInfo.WorkingPath + "\\subfolder"}",
```

Comment:
```suggestion
                    $"{XplatDll} config get all {Path.Combine(testInfo.WorkingPath, "subfolder")}",
```

Always use `Path.Combine` to combine paths, as it will use the correct path separator character for the operating system. Tehnically, `/` should work on all, but using `\` will not do what you expect on Linux, and probably Mac.